ci/laravel.yml

```diff
@@ -0,0 +1,26 @@
+name: Laravel CI
```

Comment:
Let's remove the word `CI` as per the guidelines here #156

Reply:
removed CI 
https://github.com/actions/starter-workflows/pull/138/commits/9f247c9a93ece1ceb61bfaae2c1b2b1e273560fa